otherlibs/stdune/src/stdune.ml

```diff
@@ -1,5 +1,4 @@
-[@@@alert unstable "The API of this library is not stable and may change without notice."]
-[@@@alert "-unstable"]
```

Comment:
just so we don't forget: you can work around that by passing `-alert -unstable`

Reply:
I am not familiar with alerts, I tried to add `-alert` but it still fails:
```
File "otherlibs/stdune/src/stdune.ml", line 2, characters 0-29:
2 | [@@@alert "-alert -unstable"]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error (warning 47 [attribute-payload]): illegal payload for attribute 'alert'.
Ill-formed list of alert settings

File "otherlibs/stdune/src/stdune.ml", line 1, characters 4-9:
1 | [@@@alert unstable "The API of this library is not stable and may change without notice."]
        ^^^^^
Error (warning 53 [misplaced-attribute]): the "alert" attribute cannot appear in this context
```

Reply:
taking a step back: why did you make that change? in other words, what breaks when you leave these attributes?